modules/ROOT/pages/release-notes/release-notes56.adoc

```diff
@@ -60,14 +61,14 @@ For information on the `format_empty_lines` option, see: link:{baseurl}/configur
 
 For information on:
 
-* The `TableModified` event, see: link:{baseurl}/plugins/opensource/table/#events[Table plugin - `events`].
-* Core editor events, see: link:{baseurl}/advanced/events/#editorcoreevents[Editor core events].
+* The `TableModified` event, see: xref:plugins/opensource/table.adoc#events[Table plugin - `events`].
+* Core editor events, see: xref:advanced/events.adoc#editorcoreevents[Editor core events].
```

Comment:
This xref doesn't resolve since the `editorcoreevents` id is missing in `events.adoc`. So it needs to be added there, e.g:
```
[[editorcoreevents]]
== Editor core events
```

Reply:
done